src/screens/DiscoveryScreenV2/index.js

```diff
@@ -20,11 +21,10 @@ import {
   DISCOVERY_TAB_TOPICS,
   DISCOVERY_TAB_USERS
 } from '../../utils/constants';
+import {Header} from '../../components';
 import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
 import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are just reordering of import statements. However, it's a good practice to keep the import statements organized. Usually, third-party library imports come first, followed by absolute imports and then relative imports. But there is no strict rule about this and it can vary based on the team's preference.
```diff
- import {Header} from '../../components';
- import {colors} from '../../utils/colors';
- import {fonts} from '../../utils/fonts';
- import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';

+ import {Header} from '../../components';
+ import {colors} from '../../utils/colors';
+ import {fonts} from '../../utils/fonts';
+ import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->